src/Components/Playlist.js

```diff
@@ -0,0 +1,32 @@
+import { Link } from "react-router-dom";
+import { useData } from "../Context";
+const noVideosUrl =
+  "https://pbs.twimg.com/profile_images/579325060831285248/Z76zBUF8_400x400.png";
+
+export const Playlist = ({ playlist }) => {
+  const {
+    state: { videos },
+  } = useData();
+  const getVideoImage = (videoId) => {
+    const video = videos.find((item) => item.id === videoId);
+    return video.imageUrl;
+  };
+  const playlistImage =
+    playlist.videos.length > 0
+      ? getVideoImage(playlist.videos[0])
```

Comment:
playlist image is not just a playlist.videos[0].imageUrl because If it doesn't contain any video It might result in an error. So I added a ternary to check if videos length is greater than 0.